etc/inc/disable-common.inc

```diff
@@ -512,71 +513,73 @@ blacklist ${PATH}/crontab
 blacklist ${PATH}/doas
 blacklist ${PATH}/evtest
 blacklist ${PATH}/expiry
-blacklist ${PATH}/fusermount
+blacklist ${PATH}/fping
+blacklist ${PATH}/fping6
+blacklist ${PATH}/fusermount*
 blacklist ${PATH}/gksu
 blacklist ${PATH}/gksudo
 blacklist ${PATH}/gpasswd
+blacklist ${PATH}/groupmems
+blacklist ${PATH}/hostname
+#blacklist ${PATH}/ip # breaks --ip=dhcp
 blacklist ${PATH}/kdesudo
 blacklist ${PATH}/ksu
 blacklist ${PATH}/mount
-blacklist ${PATH}/mount.ecryptfs_private
+blacklist ${PATH}/mount.*
```

Comment:
Cfr. above, IMO this is technically incorrect. I mean, there are several ${PATH}/mount.foo binaries that aren't SUID/SGID (). There might be good reasons to blacklist those too, but in that case - IMO - we should group them under a differently named section.

Reply:
> That is, I don't think that there is an easy way to list the paths of all SUID
binaries for every package on Arch, for example.

Indeed, none that I know of (not only on Arch). Would be a very nice feature if Firejail could check for SUID binaries in the sandbox and blacklist them dynamically instead of keeping track via another file list!